src/parser.ts

```diff
@@ -1292,14 +1283,12 @@ export class Parser {
         return expr;
     }
 
-    parseSuper(): Node.Super {
+    parseSuper(allowCall = false): Node.Super {
```

Comment:
That's a design pattern we'd like to avoid. Search for "boolean trap" for more details.